features/frameworks/greentea-client/greentea-client/test_env.h

```diff
@@ -28,6 +28,9 @@
 #endif
 
 #include <stdio.h>
+extern "C" {
```

Comment:
@sg- If a module expose a C and a C++ API what is the mbed way of exposing these APIs ?